supply-chain/imports.lock

```diff
@@ -0,0 +1,2 @@
+
+# cargo-vet imports lock
```

Comment:
what could be inside this lockfile ?

Reply:
Don't know, but this file is mandatory to be able to run cargo-vet.